streaming/stream_transfer_task.cc

```diff
@@ -175,6 +175,13 @@ future<> send_mutations(lw_shared_ptr<send_info> si) {
 }
 
 future<> send_mutation_fragments(lw_shared_ptr<send_info> si) {
+ return si->reader.peek(db::no_timeout).then([si] (mutation_fragment* mfp) {
+  if (!mfp) {
```

Comment:
This looks like only half-indent.

Reply:
It is on purpose to avoid massive indentation changes.

Reply:
@denesb Please take a look at the peek() usage. Looks good to you?

Reply:
> It is on purpose to avoid massive indentation changes.

Oh, I didn't see the closing parens below. Makes sense.

Reply:
> @denesb Please take a look at the peek() usage. Looks good to you?

Yes.